tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add new add-ons using maintenance test repo URLs.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi 'get_var';
+
+sub run {
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
```

Comment:
is setting the var 'MAINT_TEST_REPO' needed outside the code ?
Otherwise maybe concept can be expressed in a concise way: use "foo" if available OR "bar" if not 

```perl
my @repos = split(/,/, get_var('INCIDENT_REPO') || get_var('MAINT_TEST_REPO'));
```

and get rid of the double `get_var ` at 16

Reply:
True, but this is a direct replication of the code of the original file (see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/add_update_test_repo.pm) - and I'm not taking the risk of changing it :p

Reply:
True, but does it really help with anything? I mean, it's something that we would (probably) have to change again if we run single incidents, and it doesn't have any difference when not in single incidents. Plus, if someone decides to run single incidents with libyui, the can :p 

But I'll remove it for now.